apps/frontend-manage/src/components/activities/overview/ActivityDetailsModal.tsx

```diff
@@ -84,89 +144,367 @@ function ActivityDetailsModal({
+              <Link
+                href={'https://www.klicker.uzh.ch/gamification/grading_logic/'}
+                passHref
+                legacyBehavior
+              >
+                <a
+                  className="text-primary-100 flex flex-row items-center gap-2 text-sm font-normal hover:underline"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <FontAwesomeIcon icon={faBookOpen} />
+                  {t('manage.general.gradingDescription')}
+                </a>
+              </Link>
             </div>
-            <div>
-              {stack.elements.map((instance, instanceIx) => (
-                <Link
-                  href={`/instances/${instance.id}`}
-                  className="text-sm hover:text-slate-700"
-                  key={instance.id}
-                  legacyBehavior
-                  passHref
+          </div>
+          <Accordion type="multiple" className="w-full">
+            <RadioGroup
+              value={selectedInstance}
+              onValueChange={(val) => setSelectedInstance(val)}
+              className="m-0 w-full gap-0 p-0"
+            >
+              {stacks.map((stack, index) => (
+                <AccordionItem
+                  value={String(stack.id)}
+                  key={stack.id}
+                  className="m-0 w-full space-y-0 border-gray-300 p-2"
                 >
-                  <a
-                    data-cy={`open-instance-${instance.id}`}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    <div
-                      className={twMerge(
-                        'hover:text-primary-100 flex flex-row items-center justify-between gap-1.5 border-b text-sm',
-                        outdatedInstances.includes(instance.id)
-                          ? 'bg-uzh-red-20'
-                          : ''
-                      )}
-                      data-cy={`stack-${index}-instance-${instanceIx}`}
-                    >
-                      <div className="flex flex-row items-center gap-1.5">
-                        {outdatedInstances.includes(instance.id) ? (
+                  <div key={stack.id} className="w-full">
+                    <AccordionTrigger className="flex w-full flex-row items-center justify-between p-2 hover:no-underline">
+                      <span className="flex-1 font-bold hover:underline">
+                        {activity.type === ActivityType.LiveQuiz
+                          ? t('shared.generic.blockN', {
+                              number: index + 1,
+                            })
+                          : t('shared.generic.stackN', {
+                              number: index + 1,
+                            })}
+                      </span>
+                      <div className="flex flex-row items-center justify-end gap-3">
+                        {stack.elements.filter((element) =>
```

Comment:
idea: this can also be done by checking for stack.elements.some((element =< outdatedInstances.includes(element.instance.id))